Archipelago.HollowKnight/Archipelago.cs

```diff
@@ -275,6 +478,49 @@ private LoginResult ConnectToArchipelago()
             }
         }
 
+        private void OnDeathLinkReceived(DeathLink deathLink)
+        {
+            if (outgoingDeathlinks > 0)
+            {
+                outgoingDeathlinks--;
+                return;
+            }
+            if (deathlinkStatus == DeathLinkStatus.None)
+            {
+                deathlinkStatus = DeathLinkStatus.Pending;
+            }
+            string cause = deathLink.Cause;
+            if (cause == null || cause == "")
+            {
+                cause = $"{deathLink.Source} died.";
+            }
+            new MsgUIDef()
+            {
+                name = new BoxedString(cause),
+                sprite = new BoxedSprite(DeathLinkSprite)
+//                    GameCameras.instance.hudCamera.GetComponentsInChildren<JournalEntryStats>(true)
+//                    .Where(x => x.notesConvo == "NOTE_HOLLOW_SHADE")
+//                    .First()
+//                    .GetSprite()
+//                )
```

Comment:
Remove comments if not necessary.

Reply:
Will do for final merge, but I wanted to have this documented if we went back to it.  This was using the Shade icon from the journal entry for the death icon initially and it took a surprising amount of effort to track it down.